init.sh

```diff
@@ -25,6 +25,9 @@ debug_mode=$(echo $DEBUG)
 log_level=${LOG_LEVEL:-warn}
 marathon_host=$(echo $MARATHON_HOST)
 sleep_duration=${MARATHON_POLL_INTERVAL:-5}
+# location for a remote /etc/api-gateway folder.
+# i.e s3://api-gateway-config
+remote_config=$(echo $REMOTE_CONFIG)
```

Comment:
I've been having some issues with `/bin/sh` vs `/bin/bash` and I eventually decided to switch to `/bin/bash`. Line 48 bellow doesn't work correctly with `bin/sh` but it works correctly with `/bin/bash`. Now with this change the other form of reading vars might work too. I'll test it and then change it.